virttest/utils_v2v.py

```diff
@@ -70,6 +70,20 @@ def get_uri(self, hostname, vpx_dc=None, esx_ip=None, username=None):
         self.vpx_dc = vpx_dc
         self.esx_ip = esx_ip
         self.username = username if username else 'root'
+        self.vpx_no_username = False
+        return uri_func()
+
+    def get_uri_without_username(self, hostname, vpx_dc=None, esx_ip=None):
```

Comment:
I think keeping the logic in one function is better.
There is no need to add a new function.

Reply:
I considered keeping the logic in one function at first. 
However, this original func is for public use. If adding the parameters to control the conditions in one func, all calls in other code scripts need to be modified accordingly.
In this consideration, making the func to be isolated only for this case is safer and better.